Buildings/Experimental/DHC/EnergyTransferStations/Cooling/CoolingIndirect.mo

```diff
@@ -0,0 +1,365 @@
+within Buildings.Experimental.DHC.EnergyTransferStations.Cooling;
+model CoolingIndirect
+  "Indirect cooling energy transfer station for district energy systems"
+  extends Buildings.Fluid.Interfaces.PartialFourPortInterface(
+    redeclare final package Medium1=Medium,
+    redeclare final package Medium2=Medium,
+    final m1_flow_nominal=mDis_flow_nominal,
+    final m2_flow_nominal=mBui_flow_nominal,
+    show_T=true);
+  replaceable package Medium=Modelica.Media.Interfaces.PartialMedium
+    "Medium in the component";
+  // mass flow rates
+  parameter Modelica.SIunits.MassFlowRate mDis_flow_nominal(
+    final min=0,
+    final start=0.5)
+    "Nominal mass flow rate of primary (district) district cooling side";
+  parameter Modelica.SIunits.MassFlowRate mBui_flow_nominal(
+    final min=0,
+    final start=0.5)
+    "Nominal mass flow rate of secondary (building) district cooling side";
+  // Primary supply control valve
+  parameter Modelica.SIunits.PressureDifference dpValve_nominal(
+    final min=0,
+    final displayUnit="Pa")=6000
```

Comment:
Should final be used before displayUnit?

Reply:
No, I'll remove. Thanks